easybuild/easyconfigs/s/SWIG/SWIG-3.0.10-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,27 @@
+# @author: gppezzi, marcelsc
+
+name = 'SWIG'
+version = '3.0.10'
+pyver = '3.5.2'
+versionsuffix= '-Python-%(pyver)s'
+
+homepage = 'http://www.swig.org/'
+description = """SWIG is a software development tool that connects programs written in C and C++ with
+ a variety of high-level programming languages."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [
+    ('Python', '3.5.2'),
+    ('PCRE', '8.38'),
+]
+
+# fix for TensorFlow
```

Comment:
This was an issue on Cray  and using older version of Bazel, I've just removed the fix and it seems to work now.